content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md

```diff
@@ -6,6 +6,22 @@ menu:
     parent: About the project
 ---
 
+## v1.5.9 [TBD]
```

Comment:
Do we need to add any of the content that we pulled for the 1.5.8 release back into the 1.5./9 release?